src/genie/libs/parser/iosxe/tests/ShowSwitchStackPortsSummary/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,114 @@
+expected_output = {
```

Comment:
Please add empty test as well. like below one. just folder with empty file.
https://github.com/CiscoTestAutomation/genieparser/tree/master/src/genie/libs/parser/iosxe/tests/ShowAccessLists/cli/empty

Reply:
confirmed the change